WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -608,6 +602,18 @@ - (void)customizeAppearance
     [[WPLegacyEditorFormatToolbar appearance] setBarTintColor:[UIColor colorWithHexString:@"F9FBFC"]];
     [[WPLegacyEditorFormatToolbar appearance] setTintColor:[WPStyleGuide greyLighten10]];
     [[UIBarButtonItem appearanceWhenContainedInInstancesOfClasses:@[[WPLegacyEditorFormatToolbar class]]] setTintColor:[WPStyleGuide greyLighten10]];
+
+    // Customize the appearence of the text elements
+    [self customizeAppearanceForTextElements];
+}
+
+- (void)customizeAppearanceForTextElements
+{
+    [[UINavigationBar appearance] setTitleTextAttributes:@{NSForegroundColorAttributeName: [UIColor whiteColor], NSFontAttributeName: [WPStyleGuide fontForTextStyle:UIFontTextStyleHeadline symbolicTraits:UIFontDescriptorTraitBold]} ];
+    // Search
+    [WPStyleGuide configureSearchAppearance];
```

Comment:
You are so right! I've separeted it in two, would you mind taking another look @nheagy ?
Thanks a lot